### ISSUE TITLE
is there anyway to implement realtime monitor?

### ISSUE THREAD

**Issue by kimsreng** [cobisja/tad-php#16]
it would be awesome to have this feature.

**Comment by marianomd** [comment]
This would be a nice feature. Actually the only way I found to get logs is to download all history (slow). I would like to get only new events as they happen. I think I will have to emulate the web interface (polling /form/RTRef periodically) and see what happens.

**Comment by mahdi-reverted** [comment]
@kimsreng  yes I did it with python. It is windows application, where you can get details of member in realtime with some features of python. 
link: https://github.com/suzondas/ZKTeco-Realtime-with-Kivy
It looks like: https://www.youtube.com/watch?v=2akZ8EZWqGs&feature=youtu.be
